default/specials.txt

```diff
@@ -1969,6 +1681,43 @@ Special
 //##########################################################
 
 
+// parameter 1 is the upper case monster name
+// parameter 2 is the lower case monster name
+// parameter 3 us the monster spawn probability
+MONSTER_NEST
+'''
+        EffectsGroup
+            scope = Source
+            activation = AND [
+                Planet
+                OwnedBy affiliation = AnyEmpire
+                OwnerHasTech name = "SHP_DOMESTIC_MONSTER"
+                Random probability = @3@
+            ]
+            stackinggroup = "@1@_NEST_STACK"
+            effects = [
+                CreateShip designname = "SM_@1@_1" empire = Source.Owner
+                GenerateSitRepMessage
+                    message = "EFFECT_TAME_MONSTER_HATCHED"
+                    icon = "icons/monsters/@2@-1.png"
+                    parameters = [
+                        tag = "planet" data = Source.ID
+                        tag = "predefinedshipdesign" data = "SM_@1@_1"
+                    ]
+                    empire = Source.Owner
+            ]
+        EffectsGroup
+            scope = Source
+            activation = AND [
+                Planet
+                Turn low = 10
+                Unowned
+                Random probability = 0.12 * GalaxyMonsterFrequency / 2.0
```

Comment:
> why not 0.06 * GalaxyMonsterFrequency ?

I'm the wrong person to ask here and it's not really relevant to this PR.  Dilvish has written this in 05ffd0a57 and gave an explanation.